docs/Variables.md

```diff
@@ -0,0 +1,729 @@
+  - Target linker flags for dynamic libraries
+
+- TARGET_CFLAGS_DYNAMICLIB (str)
+  - Target compiler flags for dynamic libraries
+
+- CROSS_COMPILE (str)
+  - Enables cross-compilation for TARGET_CC, TARGET_CXX, TARGET_LD, and TARGET_STRIP on compatible host/target platform pairs
+  - Is the triple prefix for the target platform (will be prepended to the aforementioned binaries)
+
+- SDKBINPATH (str)
+  - Toolchain bin path for target platform
+
+- WATCHOS_SIMULATOR_ROOT (str)
+  - The root directory of the simulated OS
+
+- IPHONE_SIMULATOR_ROOT (str)
+  - The root directory of the simulated OS
+
+- APPLETV_SIMULATOR_ROOT (str)
+  - The root directory of the simulated OS
+
+- SUBPROJECTS (str)
+  - Directory names of subprojects to build alongside the root project
+  - Space-separated list
+
+- EXPANDED_CODE_SIGN_IDENTITY_NAME (str) **[OSX]** ----- **unused ? or blanking sys var?**
+- EXPANDED_CODE_SIGN_IDENTITY (str) **[OSX]** ----- **unused? or blanking sys var?**
+
+- LOCAL_INSTALL_PATH (str)
+  - File path where you'd like to install the final product (e.g., app, tool, library, framework, etc) (default: `XXX_INSTALL_PATH`)
+
+- LOGOS_DEFAULT_GENERATOR (str)
+  - The logos generator you'd like to use (default: `MobileSubstrate`)
+  - See https://theos.dev/docs/logos-syntax
+
+- ORION_DEFAULT_BACKEND (str)
+  - The Orion backend you'd like to use (default: `Substrate`)
+  - See https://orion.theos.dev/Enums/Backends.html
+
+- ADDITIONAL_CFLAGS (str)
+  - Additional flags passed to CC for C code
+  - Space-separated list
+
+- ADDITIONAL_CCFLAGS (str)
+  - Additional flags passed to CXX for C++ code
+  - Space-separated list
+
+- ADDITIONAL_OBJCFLAGS (str)
+  - Additional flags passed to CC for ObjC code
+  - Space-separated list
+
+- ADDITIONAL_OBJCCFLAGS (str)
+  - Additional flags passed to CXX for ObjC++ code
+  - Space-separated list
+
+- ADDITIONAL_SWIFTFLAGS (str)
+  - Additional flags passed to `swift` for Swift code
+  - Space-separated list
+
+- ADDITIONAL_LOGOSFLAGS (str)
+  - Additional flags passed to `logos` for Logos code
+  - Space-separated list
+
+- ADDITIONAL_LDFLAGS (str)
+  - Additional flags passed to linker for linking
+  - Space-separated list
+
+- ADDITIONAL_ORIONFLAGS (str)
+  - Additional flags passed during an Orion build
+  - Space-separated list
+
+- ADDITIONAL_XCODEFLAGS (str) **[OSX]**
+  - Additional flags passed to `xcodebuild`
+  - Space-separated list
+
+- ADDITIONAL_XCODEOPTS (str) **[OSX]**
+  - Additional options passed to `xcodebuild`
+  - Space-separated list
+
+- USE_DEPS (bool)
+  - Toggle dependency tracking (i.e., makedeps) (default: `0`)
+  - Compiler writes headers to $THEOS_OBJ_DIR/*.Td
+
+- MDFLAGS (str)
+  - Flags passed for makedeps
+  - Space-separated list
+
+- LOCAL_BUNDLE_NAME (str)
+  - Name for the current project instance's bundle
+
+- LOCAL_BUNDLE_EXTENSION (str)
+  - File extension for the current project instance's bundle
+
+- `ALL_*FLAGS` (str) ----- **internal ?**
```

Comment:
Do we want to include these or are we considering them "internal" and not to be outward facing?

Reply:
If it's internal we might not want to document it in a the public version as it might change without notice or care for backwards compatibility